client/main.js

```diff
@@ -376,6 +376,37 @@ $(document).ready(function() {
     }
   }
 
+  $('#nav-wiki-btn').on('click', toggleWiki);
+
+  $('.wiki-aside-action-collapse').on('click', collapseWiki);
+
+  function showWiki() {
+    if (!main.isWikiAsideLoad) {
+      var lang = window.location.toString().match(/\/\w{2}\//);
+      lang = (lang) ? lang[0] : '/en/';
+      var wikiURL = 'http://freecodecamp.github.io/wiki' + lang;
+      var wikiAside = $('<iframe>');
+      wikiAside.attr('src', wikiURL);
+      $('.wiki-aside').append(wikiAside);
+      main.isWikiAsideLoad = true;
+    }
```

Comment:
See the `if(!main.isWikiAsideLoad)` and the `main.isWikiAsideLoad = true`?  That should cause only one iframe to be created.  This is literally identical code to the map toggle which we are using - I cut, pasted, and did a search and replace for the word `map`.